vendor/eol-compat/dune

```diff
@@ -0,0 +1,3 @@
+(library
+ (name eol_compat)
```

Comment:
Why is this placed in the `vendor` directory ? Would be better in `lib`.

Reply:
done